desafio-03/KarineFBrito/java/Palindromo.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Palindromo {
+  public static void main(String[] args) {
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Escolha o número inicial: ");
+    int n1 = sc.nextInt();
+    System.out.println("Escolha o numero final: ");
+    int n2 = sc.nextInt();
```

Comment:
Na descrição da desafio fala que não pode utilizar números negativos, o teu programa está fazendo tratamento para isto?

Segundo ponto: o que acontece se o primeiro número for maior que o segundo? Tem tratamento para isto